mocha.js

```diff
@@ -7430,7 +7430,7 @@ function slowToString (encoding, start, end) {
     return ''
   }
 
-  // Force coersion to uint32. This will also coerce falsey/NaN values to 0.
+  // Force coersion to uint32. This will also coerce falsy/NaN values to 0.
```

Comment:
I'm not so sure that "falsey" is incorrect -- niether spelling should show up in a normal spellchecker/dictionary as it's programming jargon, but I've definitely seen the spelling with an E used.

Reply:
Both are used in this file. This spelling is only used once in the entire repository.

Reply:
Hrm, by ignoring the mocha.js, the thing goes away, which means that it's coming from some included module. If you happen to know which module, I can fix it there. Otherwise, this will disappear as I rebase back/forth...

Reply:
> Both are used in this file. This spelling is only used once in the entire repository.

Huh, interesting catch -- consistency is definitely a plus then. Well, maybe less so for the package it comes from if it's the only use of the term there, but...

Speaking of which, I may or may not be able to track down which project it's from... It's generally a matter of scrolling up (or is it down? I forget) in the bundled file till you find the name in the bundle glue (the beginning of which will be the package, but the full module path should be in there), then `npm view <package name>` and looking for GitHub URLs. But if there aren't any GitHub URLs for the package, it gets harder...